alma/lib/Forms/ShareOfCheckoutAdminFormBuilder.php

```diff
@@ -37,14 +37,21 @@ class ShareOfCheckoutAdminFormBuilder extends AbstractAlmaAdminFormBuilder
 {
     const ALMA_ACTIVATE_SHARE_OF_CHECKOUT = 'ALMA_ACTIVATE_SHARE_OF_CHECKOUT';
     const ALMA_SHARE_OF_CHECKOUT_DATE = 'ALMA_SHARE_OF_CHECKOUT_DATE';
+    const ALMA_SHARE_OF_CHECKOUT_CONSENT_NO_ANSWER = 'no_answer';
+    const ALMA_SHARE_OF_CHECKOUT_CONSENT_YES = '1';
+    const ALMA_SHARE_OF_CHECKOUT_CONSENT_NO = '0';
```

Comment:
```suggestion
    const ALMA_SHARE_OF_CHECKOUT_CONSENT_UNSET = 'unset';
    const ALMA_SHARE_OF_CHECKOUT_CONSENT_YES = 'yes';
    const ALMA_SHARE_OF_CHECKOUT_CONSENT_NO = 'no';
```

I don't really like `no_answer` and I prefer having string everywhere (even if `'1'` is actually considered as a string)

If this suggestion is not too expensive in time, I would like:
* at least the constant becomes `ALMA_SHARE_OF_CHECKOUT_CONSENT_UNSET`
* and if possible to change the value of `*_YES` and `*_NO` as suggested with two strings (`'yes'` and `'no'`)

Reply:
@syjust-alma 
I tried to use 'yes' and 'no' but I couldn't because the getter Prestashop convert the 'no' in 'true' 😔
I need 0 for false in my builder form.
I changed the name NO_ANSWER anyway